ultraLatency.py

```diff
@@ -182,6 +190,7 @@
         pass
     configureScanModule(ohboard, options.gtx, mode, mask,
                         scanmin=LATENCY_MIN, scanmax=LATENCY_MAX,
+                        stepsize=step,
```

Comment:
`$DATA_PATH` is a `$SHELL` variable that you set.  You can change the directory it is set too, and based on my earlier comments you should.